lib/core/versions/latest/RequestHandler.ts

```diff
@@ -189,11 +180,26 @@ export default class RequestHandler implements IRequestHandler {
 
     // The code reaches here if this DID is not registered on the ledger.
 
-    didDocument = await Document.fromLongFormDid(did);
+    const document = await this.applyCreateOperation(did.createOperation!);
```

Comment:
This is not a question specific to this pr, but this pr got me thinking how I'd use long form, so I thought of this. 

What is preventing me from just always use long form so I don't have to pay the fee to put my document onto a ledger? So say I make a valid operationRequest paying the bare minimum fee to avoid cost, that will allow me to keep using my long form and never having to worry if my document is actually anchored.

Reply:
This is correct - you can instantly use the DIDs you generate via their long form ID variant, and continue doing so until you need to perform a DID Doc modification (roll keys, change endpoints, etc.). Instant use and registration load reduction were the two main reasons we created long form ID variants in the first place. IDs for some types of things (i.e. inanimate objects, IOT devices) may never be registered at all, and go their entire service lifetimes using the long form ID variant.